lib/network/modules/NodesHandler.js

```diff
@@ -285,7 +287,9 @@ class NodesHandler {
       let data = changedData[i];
       if (node !== undefined) {
         // update node
-        dataChanged = node.setOptions(data);
+        if (node.setOptions(data) === true) {  // explicit test because setOptions() can return undefined
```

Comment:
OK, I've convinced myself that the handling in case of `undefined` is good. Adjusting it.